scripts/config

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source /usr/share/yunohost/helpers
+
+# Stop script if errors
+ynh_abort_if_errors
+
+# Import common fonctions
+source ./psql.sh
+source ./experimental_helper.sh
+source ./_common.sh
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+app=$YNH_APP_INSTANCE_NAME
```

Comment:
Here I get
```
./config: ligne 23: YNH_APP_INSTANCE_NAME : variable sans liaison
```
and it works much better with `$YNH_APP_ID`... :sweat_smile:

Reply:
@maniackcrudelis  ?

Reply:
@maniackcrudelis , @Psycojoker do you know why $YNH_APP_INSTANCE_NAME is not defined ?

Reply:
@Josue-T was it working for you...?

Reply:
Didn't tested after this change because I trusted @maniackcrudelis :smile:

Reply:
Indeed, I reproduced that on my own apps as well.
Still, using $YNH_APP_ID for a multi-instance app isn't a good idea.

Reply:
A fix will arrive, meanwhile I'm using this temporary patch:
```
app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
```

Reply:
Is this fix released in yunohost 3.2.1 ?

Reply:
Don't think so, @Psycojoker did you had time to fix that ?

Reply:
I tested a new time. And now it work on my side.

So it should be fixed.